### ISSUE TITLE
Disable "Home" and "End" hotkeys

### ISSUE THREAD

**Issue by tadashi-aikawa** [tadashi-aikawa/obsidian-another-quick-switcher#160]
Since [Obsidian v1.2.0](https://forum.obsidian.md/t/obsidian-release-v1-2-0-insider-build/57173), "Home" and "End" keys are assigned to faster keyboard navigations. But I think it should be disabled by default because

1. some keys are assigned to the same feature ("page up" and "page down" keys)
2. "Home" and "End" keys are used by moving the input cursor to begin or end quickly

**Comment by tadashi-aikawa** [comment]
Released in [v8.8.0](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/8.8.0) 🚀
